### ISSUE TITLE
Create UI for optimistic recipient registry to remove projects

### ISSUE THREAD

**Issue by samajammin** [clrfund/monorepo#343]
Similar to https://github.com/clrfund/monorepo/issues/295, it'd be nice to have a UI for users to request removal of recipients.

- Ability to submit `removeRecipient` tx to the registry contract, e.g. a "Report this project" button on the project page.
- Ability to share evidence with the registry owner (to either prove you are the recipient & want to be removed or that the recipient is somehow breaking the rules, e.g. by bribing contributors). This evidence is needed for the owner to determine if the removal request is legitimate. This could be a form or simple instructions to email the evidence somewhere / report the evidence on the CLR.fund forum.

**Comment by samajammin** [comment]
This is something @ryancreatescopy & I could potentially help out with.

**Comment by ryancreatescopy** [comment]
agreed, I think this is something we'll end up designing as part of our work. Happy to share mockups when I'm done.
